models/schedule_tabulate.go

```diff
@@ -54,6 +54,14 @@ func straddlesDay(s *ScheduleItem) bool {
 	return straddle
 }
 
+func listStraddlesDay(s *ScheduleItem) bool {
+	dayBoundary := 0
+	adjustedStart := s.Start.Add(time.Hour * time.Duration(-dayBoundary))
+	adjustedEnd := s.Finish.Add(time.Hour * time.Duration(-dayBoundary))
```

Comment:
Bit confused by this - why is there a variable that's always `0`, and why are we adding zero hours to the start/finish time?

Reply:
I copied the function from above and modified it badly. I can fix it if needed. Tbh I might rewrite the schedule, the code for it is a mess